variants/STM32L0xx/L011K(3-4)U_L021K4U/generic_clock.c

```diff
@@ -21,8 +21,39 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
```

Comment:
```suggestion
  RCC_OscInitTypeDef RCC_OscInitStruct = {};
  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
```

Reply:
@fpistm Sorry I keep forgetting this, can a note for this and the Astyle formating requirement be added to the ["Add-a-new-variant-(board)" wiki](https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-(board))?

Would you like me to make a pull request for that modification to the Docs?

Reply:
No need to make a PR. Simply edit the page. Rational behind this is to avoid warning at compile time.

Reply:
I've added a note: https://github.com/stm32duino/wiki/wiki/Add-a-new-variant-(board)#3---generic-system-clock-configuration